ckan/logic/action/get.py

```diff
@@ -1261,7 +1264,7 @@ def group_show(context, data_dict):
          (optional, default: ``True``)
     :type include_groups: bool
     :param include_tags: include the group's tags
-         (optional, default: ``True``)
+         (optional, default: ``False``)
```

Comment:
https://github.com/ckan/ckan/pull/4215/files#diff-4494aaf212251212949348df94158668R1186 - it still set to true

Reply:
i guess its by mistake, didnt notice  it when i was doing the changes. Thanks!